src/D2L.CodeStyle.Analyzers/Extensions/Microsoft.CodeAnalysis.CSharp.Syntax.cs

```diff
@@ -55,8 +56,19 @@ INamedTypeSymbol attributeType
 			return true;
 		}
 
-		public static bool IsFromDocComment(
-			this SyntaxNode node
-		) => node.FirstAncestorOrSelf<DocumentationCommentTriviaSyntax>() != null;
+		public static bool IsFromDocComment( this SyntaxNode node )
+			=> node.FirstAncestorOrSelf<DocumentationCommentTriviaSyntax>() != null;
+
+		public static bool IsStaticFunction( this ExpressionSyntax expr ) {
+			var afes = expr as AnonymousFunctionExpressionSyntax;

```

Comment:
This is the base class of `AnonymousMethodExpressionSyntax` and `LambdaExpressionSyntax` (remember this for the StatelessFunc analyzer below)